app/scripts/modules/core/src/artifact/react/StageArtifactSelectorDelegate.tsx

```diff
@@ -3,23 +3,21 @@ import React from 'react';
 import { StageConfigField } from 'core/pipeline';
 
 import { IStageArtifactSelectorProps, StageArtifactSelector } from './StageArtifactSelector';
-import { IPreRewriteArtifactSelectorProps, PreRewriteStageArtifactSelector } from './PreRewriteStageArtifactSelector';
-import { ArtifactsMode, ArtifactsModeService } from '../ArtifactsModeService';
 
 interface IStageArtifactSelectorDelegateProps {
   helpKey?: string;
   label: string;
   fieldColumns?: number;
 }
 
+// Please consider this component deprecated. Use StageArtifactSelector
+// directly instead.
```

Comment:
This tag will cause the deprecation to show up in IDEs

```suggestion
/** @deprecated use StageArtifactSelector instead */
```

Reply:
Great suggestion, thanks for making it inline so I could use the handy "commit suggestion" button 🌮